website/bin/exr2aces.rst

```diff
@@ -15,26 +15,46 @@ Description
 Read an OpenEXR file from infile and save the contents
 in ACES image file outfile.
 
-The ACES image file format is a subset of the OpenEXR file
-format.  ACES image files are restricted as follows:
+The ACES Image Container File Format specified in `SMPTE ST 2065-4`_ 
+is a subset of the OpenEXR file format defined by the implementation 
+of the time (roughly OpenEXR 1.6). 
 
-- Images are stored as scanlines; tiles are not allowed.
+This imposes considerable restrictions on what image data and metadata 
+can be transported from a modern OpenEXR file into an ACES Image 
+Container File, including (but not limited to) the following:
 
-- Images contain three color channels, either R, G, B (red, green,
-  blue) or Y, RY, BY (luminance, sub-sampled chroma)
+- The version field must be either be 2 or 1026; a value of 2 implies 
+  attribute names cannot be longer than 31 characters in length, 
+  as was the case prior to OpenEXR 1.7.
 
-- Images may optionally contain an alpha channel.
+- Images must be stored as scanlines; tiles are not allowed.
 
-- Only three compression types are allowed:
+- Images must contain RGB and possibly an A channel if they are monoscopic; 
+  images representing stereo pairs would add another three (or four if A is 
+  present) channels. Other channels are not permitted, including channels 
+  that otherwise might represent a combination of a luminance channel and 
+  chromaticity channels.
 
-  * NO_COMPRESSION (file is not compressed)
+- The image must not be compressed.
 
-  * PIZ_COMPRESSION (lossless)
+- The `chromaticities` attribute must specify the ACES RGB primaries and 
+  the ACES neutral as specified in SMPTE ST 2065-1.
 
-  * B44A_COMPRESSION (lossy)
+- The `acesImageContainerFlag` flat must be present and have the value 1.
 
-- The "chromaticities" header attribute must specify
-  the ACES RGB primaries and white point.
+For the full set of restrictions, see `SMTPE ST 2065-4`_ (or any 
```

Comment:
```suggestion
For the full set of restrictions, see `SMTPE ST 2065-4  <https://doi.org/10.5594/SMPTE.ST2065-4.2013>`_ (or any 
```

Sphinx is barfing on the reference, possibly because it has spaces? Not sure, but putting the url inline with the link seems to work.

Then remove the link at the bottom of the file.